waiter/src/waiter/state/maintainer.clj

```diff
@@ -642,6 +650,12 @@
                                                        (if (seq new-instance-ids) (str "New healthy instances: " new-instance-ids ".") "")
                                                        (if (seq rem-instance-ids) (str "Removed healthy instances: " rem-instance-ids ".") "")
                                                        (if (seq unhealthy-instance-ids) (str "Unhealthy instances: " unhealthy-instance-ids ".") ""))))
+                                         (when (not= (get service-id->expired-instances service-id) expired-instances)
```

Comment:
Since this logic is only used for InstanceTracker logging, let's predicate this extra work on the logger actually being enabled.

https://clojure.github.io/tools.logging/#clojure.tools.logging/enabled?

Reply:
Done